src/Shared.Core/Utilities/Tablet.cs

```diff
@@ -6,50 +6,230 @@
 namespace KKAPI
 {
 #pragma warning disable CS1591
-
     /// <summary>
-    /// Represents a data packet structure used for interacting with tablet hardware,
-    /// containing information about button states, positional coordinates, and
-    /// pressure sensitivity. This structure is primarily utilized for processing
-    /// input data from digitizing devices.
+    /// Represents a Wintab packet containing tablet input data such as position, pressure, and orientation.
     /// </summary>
+    /// <remarks>
+    /// <para>
+    /// This structure maps directly to the Wintab PACKET structure and must maintain sequential layout
+    /// with single-byte packing for correct COM interop with the Wintab driver.
+    /// </para>
+    /// <para>
+    /// The fields included in this packet are determined by the <c>lcPktData</c> mask specified when
+    /// opening the tablet context. Only fields corresponding to bits set in that mask will contain
+    /// valid data.
+    /// </para>
+    /// </remarks>
+    /// <seealso href="https://developer-docs.wacom.com/intuos-cintiq-business-702/docs/wintab-packet"/>
     [StructLayout(LayoutKind.Sequential, Pack = 1)]
     public struct Packet
     {
-        public IntPtr pkContext;        		// PK_CONTEXT
-        public uint pkStatus;           		// PK_STATUS
-        public uint pkTime;             		// PK_TIME
-        public uint pkChanged;          		// PK_CHANGED
-        public uint pkSerialNumber;     		// PK_SERIAL_NUMBER
-        public uint pkCursor;           		// PK_CURSOR
-        public uint pkButtons;          		// PK_BUTTONS
-        public int pkX;                 		// PK_X
-        public int pkY;                 		// PK_Y
-        public int pkZ;                 		// PK_Z
-        public uint pkNormalPressure;   		// PK_NORMAL_PRESSURE
-        public uint pkTangentPressure;  		// PK_TANGENT_PRESSURE
-        public Orientation pkOrientation;       // PK_ORIENTATION
-        public Rotation pkRotation;             // PK_ROTATION
+        /// <summary>
+        /// Handle to the tablet context that generated this packet.
+        /// </summary>
+        public IntPtr Context;
```

Comment:
All of those renames are breaking changes too since some of those classes are public. Are you sure no plugin used this API at all?